app/Services/MapboxGeocodingService.php

```diff
@@ -14,10 +14,11 @@ class MapboxGeocodingService
 
     public function getCoordinatesFromApi(string $cityName, string $countryName, $client = new Client()): array
     {
-        $token = config('app.mapbox_token');
+        $token = config("app.mapbox_token");
```

Comment:
Maybe create `mapbox.php` in config dir and put token here? Also API URL can be put here.

Reply:
Sure